### DISCUSSION TITLE
Flawless on Android Phone, Stuttering on TV

### DISCUSSION THREAD

**Discussion by Blickwinkel145** [LizardByte/community#267] [Sunshine] [answered]
I have my PC connected by cable to my router, which has a network setup for 5 ghz.
I ran sunshine on default settings, and while on my phone (streaming to which ironically I don't care about) it runs flawlessly on very high bitrate and settings, my TV which is a google TV basically gets hiccups. 

"Frames dropped by your network connection" jump between 4-7% and I basically get a second of okay frames, second of stutter, second of okay frames and then stutter, and so on. Audio is pulsing with it.

It can't be a probem with connectivity, the phone held in the spot TV is placed does just fine.
Tried all the settings I could think of, turned off bluetooth on TV, lowest possible quality, nothing changes.
Is it possible the TV just can't handle the moonlight app? It's a TCL c635 from 2022 though.

**Comment by ReenigneArcher** [comment, marked as answer]
Generally speaking, TV manufacturers really cheap out on the network cards. If you can add an external one I'd be willing to bet the situation would greatly improve.

Hardwiring is always preferred over WiFi also.

**Reply by Blickwinkel145** [reply]
I wish there was a place to just screw in an external antenna, first thing I would have tried. 
I guess I'm gonna buy a really long lan cable and connect the tv across the room just to see if that even changes anything.

**Comment by Blickwinkel145** [comment]
Okay, so I've tried the cable and it works just fine, but having a cable hanging on ankle height through the middle of the living room is not really a solution. So is there any way to fix this without buying a new TV? Anything that works with google tv that would boost the wireless capability?
